certifikit/src/commonMain/kotlin/app/cash/certifikit/okio/ForwardingSource.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
```

Comment:
What’s with these compacted copyright statements? Can you do ’em regular instead?

Reply:
I've tried to automate this in various projects, and messed up config.  Fixed.